patches/minecraft/net/minecraft/world/entity/Entity.java.patch

```diff
@@ -414,6 +432,33 @@
           });
        }
  
+@@ -2559,7 +_,7 @@
+    }
+ 
+    protected AABB m_20217_(Pose p_20218_) {
+-      EntityDimensions entitydimensions = this.m_6972_(p_20218_);
++      EntityDimensions entitydimensions = this.getDimensionsForge(p_20218_);
+       float f = entitydimensions.f_20377_ / 2.0F;
+       Vec3 vec3 = new Vec3(this.m_20185_() - (double)f, this.m_20186_(), this.m_20189_() - (double)f);
+       Vec3 vec31 = new Vec3(this.m_20185_() + (double)f, this.m_20186_() + (double)entitydimensions.f_20378_, this.m_20189_() + (double)f);
+@@ -2570,12 +_,16 @@
+       this.f_19828_ = p_20012_;
+    }
+ 
++   /**
++    * @deprecated Can be overridden. Use {@link #getEyeHeightForge(Pose, EntitySize)} instead.
++    */
++   @Deprecated
```

Comment:
May be better to read
"Should be overriden, but callers should invoke `#getEyeHeightForge(Pose, Dims)` to compute the true eye height.  If you do not want to compute a new value, use `#getEyeHeight()`"